ajax/libs/darkroomjs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "darkroomjs",
+  "description": "Extensible image editing tool via HTML canvas",
+  "version": "2.0.1",
+  "filename": "darkroom.js",
+  "license": "MIT",
+  "homepage": "https://mattketmo.github.io/darkroomjs",
+  "keywords": [
+    "image",
+    "canvas",
+    "crop"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mattketmo/darkroomjs.git"
```

Comment:
Let's use `https://github.com/Mattketmo/darkroomjs.git` to keep the consistency, thanks!